bookwyrm/templates/layout.html

```diff
@@ -127,17 +127,19 @@
                     <div class="column">
                         <form name="login" method="post" action="/user-login">
                             {% csrf_token %}
-                            <div class="field is-grouped">
-                                <div class="control">
+                            <div class="columns is-variable is-1">
```

Comment:
Narrowing the column gap here isn't totally necessary (and will only work on some browsers anyway), but with columns the gap was wider than it used to be, and now it's narrower than it used to be. I thought narrower was a little better, but if wider is better we can drop this and use the default gap.